### ISSUE TITLE
the error module 'FirebaseFirestore' not found" when run "pod install"

### ISSUE THREAD

**Issue by v-anh** [firebase/firebase-ios-sdk#1287]
* Xcode version: 9.4
  * Firebase SDK version: 5.0
  * Firebase Component: Firestore
  * Component version: 0.12.1

Pod file contains:
pod 'Firebase/Firestore'

I receive the error module 'FirebaseFirestore' not found" when I add plugins cordova-plugin-firestore to the project.
Also, when I run "pod install", I don't see FirebaseFirestore.framework file in ./Pods/FirebaseFirestore/...

**Comment by paulb777** [comment]
@v-anh We'll likely need more information to reproduce. Adding `pod 'Firebase/Firestore'` to a Podfile works fine for me.  Are you able to share the full Podfile?  If the issue is Cordova related, it may be helpful to report the issue in that community.

**Comment by v-anh** [comment]
Thanks to you for supporting me.
This is full Podfile:
platform :ios, '8.0'
target 'My-App' do
	project 'My-App.xcodeproj'
	pod 'Firebase'
	pod 'Firebase/Core'
	pod 'Firebase/Firestore'
end

Full Podfile.lock:

`PODS:
  - BoringSSL (10.0.2):
    - BoringSSL/Implementation (= 10.0.2)
    - BoringSSL/Interface (= 10.0.2)
  - BoringSSL/Implementation (10.0.2):
    - BoringSSL/Interface (= 10.0.2)
  - BoringSSL/Interface (10.0.2)
  - Firebase (5.0.1):
    - Firebase/Core (= 5.0.1)
  - Firebase/Core (5.0.1):
    - Firebase/CoreOnly
    - FirebaseAnalytics (= 5.0.0)
  - Firebase/CoreOnly (5.0.1):
    - FirebaseCore (= 5.0.1)
  - Firebase/Firestore (5.0.1):
    - Firebase/CoreOnly
    - FirebaseFirestore (= 0.12.1)
  - FirebaseAnalytics (5.0.0):
    - FirebaseCore (~> 5.0)
    - FirebaseInstanceID (~> 3.0)
    - "GoogleToolboxForMac/NSData+zlib (~> 2.1)"
    - nanopb (~> 0.3)
  - FirebaseCore (5.0.1):
    - "GoogleToolboxForMac/NSData+zlib (~> 2.1)"
  - FirebaseFirestore (0.12.1):
    - FirebaseCore (~> 5.0)
    - FirebaseFirestore/abseil-cpp (= 0.12.1)
    - gRPC-ProtoRPC (~> 1.0)
    - leveldb-library (~> 1.18)
    - Protobuf (~> 3.1)
  - FirebaseFirestore/abseil-cpp (0.12.1):
    - FirebaseCore (~> 5.0)
    - gRPC-ProtoRPC (~> 1.0)
    - leveldb-library (~> 1.18)
    - Protobuf (~> 3.1)
  - FirebaseInstanceID (3.0.0):
    - FirebaseCore (~> 5.0)
  - GoogleToolboxForMac/Defines (2.1.4)
  - "GoogleToolboxForMac/NSData+zlib (2.1.4)":
    - GoogleToolboxForMac/Defines (= 2.1.4)
  - gRPC (1.12.0):
    - gRPC-RxLibrary (= 1.12.0)
    - gRPC/Main (= 1.12.0)
  - gRPC-Core (1.12.0):
    - gRPC-Core/Implementation (= 1.12.0)
    - gRPC-Core/Interface (= 1.12.0)
  - gRPC-Core/Implementation (1.12.0):
    - BoringSSL (~> 10.0)
    - gRPC-Core/Interface (= 1.12.0)
    - nanopb (~> 0.3)
  - gRPC-Core/Interface (1.12.0)
  - gRPC-ProtoRPC (1.12.0):
    - gRPC (= 1.12.0)
    - gRPC-RxLibrary (= 1.12.0)
    - Protobuf (~> 3.0)
  - gRPC-RxLibrary (1.12.0)
  - gRPC/Main (1.12.0):
    - gRPC-Core (= 1.12.0)
    - gRPC-RxLibrary (= 1.12.0)
  - leveldb-library (1.20)
  - nanopb (0.3.8):
    - nanopb/decode (= 0.3.8)
    - nanopb/encode (= 0.3.8)
  - nanopb/decode (0.3.8)
  - nanopb/encode (0.3.8)
  - Protobuf (3.5.0)

DEPENDENCIES:
  - Firebase
  - Firebase/Core
  - Firebase/Firestore

SPEC REPOS:
  https://github.com/cocoapods/specs.git:
    - BoringSSL
    - Firebase
    - FirebaseAnalytics
    - FirebaseCore
    - FirebaseFirestore
    - FirebaseInstanceID
    - GoogleToolboxForMac
    - gRPC
    - gRPC-Core
    - gRPC-ProtoRPC
    - gRPC-RxLibrary
    - leveldb-library
    - nanopb
    - Protobuf

SPEC CHECKSUMS:
  BoringSSL: 60dd24df4af296bf41d78e5841dbb95d75f88c0d
  Firebase: d6861c2059d8c32d1e6dd8932e22ada346d90a3a
  FirebaseAnalytics: 19812b49fa5f283dd6b23edf8a14b5d477029ab8
  FirebaseCore: cafc814b2d84fc8733f09e653041cc2165332ad7
  FirebaseFirestore: f686b8e83f3cf8bbc37db6e98e01029a14f01f55
  FirebaseInstanceID: 83e0040351565df711a5db3d8ebe5ea21aca998a
  GoogleToolboxForMac: 91c824d21e85b31c2aae9bb011c5027c9b4e738f
  gRPC: 9362451032695e2dfb7bafcd3740e3a27939e4ff
  gRPC-Core: 9696b220565b283e021cf2722d473a4a74b7622a
  gRPC-ProtoRPC: a1bd56fb1991a8dae4581250d7259eddabb66779
  gRPC-RxLibrary: 1ed5314e8b38cd6e55c9bfa048387136ae925ce9
  leveldb-library: 08cba283675b7ed2d99629a4bc5fd052cd2bb6a5
  nanopb: 5601e6bca2dbf1ed831b519092ec110f66982ca3
  Protobuf: 8a9838fba8dae3389230e1b7f8c104aa32389c03

PODFILE CHECKSUM: 2f8e274c641210e30bf60cabf5e76f95a840a484

COCOAPODS: 1.5.2`

And I don't see FirebaseFirestore.framework file in ./Pods/FirebaseFirestore/... in that, I just see Firestore folder.
![screen shot 2018-05-18 at 9 17 02 am](https://user-images.githubusercontent.com/5137045/40212870-902b536c-5a7c-11e8-949a-4d558e402ad8.png)

Thanks.

**Comment by paulb777** [comment]
tl;dr 
Change to `@import Firebase`;

I suspect that your Podfile does not include `use_frameworks!`.   Therefore, CocoaPods would not create a FirebaseFirestore module now that with the Firebase 5.0.0 release, FirebaseFirestore is a source pod. 

The recommended way to import Firebase modules is with `@import Firebase`. If you do that, instead of `@import FirebaseFirestore`, the import will work with both Podfiles that include and do not include `use_frameworks!`.

**Comment by v-anh** [comment]
@paulb777  Thank you so much. I fixed the error module 'FirebaseFirestore' not found". But I receive the new error about "FIRQueryListenOptions" type. I think maybe "cordova-plugin-firestore" has the old code for the Firebase 5.0.0 release. What do you think about this?
![screen shot 2018-05-18 at 4 23 35 pm](https://user-images.githubusercontent.com/5137045/40227492-e4d50930-5ab8-11e8-88e0-23759ead5f5e.png)

**Comment by wilhuff** [comment]
The 5.0.0 release did indeed make those changes: https://firebase.google.com/support/release-notes/ios

**Comment by v-anh** [comment]
@paulb777 @wilhuff Thank you so much.
